CMakeLists.txt

```diff
@@ -2382,21 +2361,11 @@ IF(HAVE_BUILD_ID_SUPPORT AND NOT WITHOUT_SERVER)
 ENDIF()
 
 UNSET(MYSQLD_EXECUTABLE_FOR_INFO_BIN)
-IF(HAVE_BUILD_ID_SUPPORT AND NOT WITHOUT_SERVER)
+IF(HAVE_BUILD_ID_SUPPORT AND NOT WITHOUT_SERVER AND NOT WITH_XTRABACKUP)
   SET(MYSQLD_EXECUTABLE_FOR_INFO_BIN
     "-DMYSQLD_EXECUTABLE=$<TARGET_FILE:mysqld>")
 ENDIF()
 
-# Build flags: This must be postponed to the make phase.
```

Comment:
better to add `IF(NOT WITH_XTRABACKUP)` here rather than deleting. In case they change anything on those lines in the future we wont get conflicts.

Reply:
fixed